File: easy/countDigits.java
class countDigits{
 
   public int countDigits(int num) {

        int keep = num ;
        
        int count = 0 ; 

        while ( num >0){
            int val = num % 10 ;

            if( keep % val ==0){
                count++ ;
            }

            num = num / 10 ;

        }
        return count ; 
        
    } 

}

File: easy/count-Digits.rs

    pub fn count_digits(mut num: i32) -> i32 {

        let mut count : i32 = 0 ;
        let original = num ;

        while ( num > 0){

            let mut val = num % 10 ; 

            if (original % val ==0 && val != 0  ){
                count+=1;
            } 
            num = num /10; 
        } 
        count 
    }
